cogs/auto_standup.py

```diff
@@ -0,0 +1,45 @@
+import discord
+import asyncio
+import datetime
+from datetime import timezone
+from discord import app_commands
+from discord.ext import commands
+
+channel_ID = 1172727647382552737
+day_seconds = 86400
+timezone_offset = 11
+thursday = 3
+hour = 9
+
+
+class AutoStandups(commands.Cog):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.channel_id = channel_ID
+        self.time_offset = datetime.timedelta(hours=timezone_offset)
+        self.bot.loop.create_task(self.send_weekly_message())
+
+    async def send_weekly_message(self):
+        await self.bot.wait_until_ready()
+        channel = self.bot.get_channel(self.channel_id)
+        while not self.bot.is_closed():
+            utc_time = datetime.datetime.now(timezone.utc)
+            curr_time = utc_time + self.time_offset
+
+            if curr_time.weekday() == thursday and curr_time.hour == hour:
```

Comment:
Instead of `thursday`, to avoid it seeming like it is hardcoded for the future, would change this to variable name to `day`